grails-app/i18n/messages.properties

```diff
@@ -3368,11 +3368,14 @@ react.inboundReturns.confirmPreviousPage.message=Cannot save due to validation e
 
 productsConfiguration.label=Categories and Products Configuration
 productsConfiguration.unspscCategories.label=UNSPSC category list
+productsConfiguration.whoCategories.label=WHO category list
 productsConfiguration.defaultCategoryTree.label=Default Openboxes category tree
+productsConfiguration.whoProducts.label=WHO product list
```

Comment:
I don't think we need any of these (WHO, UNSPSC) to be translatable because they are configurable.

Reply:
@jmiranda since we are not building it from config yet, then it can stay for now. The dev that will refactor the front part to be built based on the config should remove it.

Unless @pmuchowski you want to make this change in this pr.